### PR TITLE
Improve callback waterfalls of cli scripts

### LINKED ISSUE #406: Cleanup callback waterfalls in cli scripts
## Are you reporting a bug or a feature?

 - [ ] Bug
 - [x] Feature

## Expected Behavior

I expect easy to read, step-by-step, sequential user interaction of our CLI scripts.

## Actual Behavior

Currently our cli scripts like `release.js` develop huge callback waterfalls.

### PR DESCRIPTION
**Pull request by AndyOGo**
Fixes #406 

Changes proposed in this pull request:

 - NPM installed `async-waterfall` and `chalk`
 - reduced callback waterwalls to level of 2
 - fixes cross os terminal colors

## Type of change

- [x] New feature (non-breaking change which adds functionality)
 
 # How Has This Been Tested?

Not yet, we will need a release for it...
 
 
 
 # Checklist:
 
 - [x] My code follows the style guidelines of this project
 - [x] I have performed a self-review of my own code
 - [x] I have commented my code, particularly in hard-to-understand areas
 - [x] I have made corresponding changes to the documentation
 - [x] My changes generate no new warnings
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: stack/tasks/release.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const outdent = require('outdent');
 const { exec } = require('child_process');
+
```
Comment (LucaMele) [inline stack/tasks/release.js]: Is this all of the pr?

Comment (AndyOGo) [reply stack/tasks/release.js]: hehe no, I had to stop and work on other stuff

**Comment by AndyOGo** [comment]
@LucaMele 
Ready for review

File: stack/tasks/release.js [hunk 2]
```diff
@@ -159,93 +171,68 @@ const confirmedRelease = (type, version) => {
...
+  waterfall([
+    (callback) => {
+      exec(`git checkout ${developTrunk} && git pull && git checkout -b release-tmp`, handleSuccess(callback, () => {
+        console.log(chalk.cyan(outdent`
           Step 1 complete...
-        `,
+        `));
+      }));
+    },
+    (callback) => {
```
[the remark this answers is not in the document]
Comment (AndyOGo) [reply stack/tasks/release.js]: Found it, it's called with `error`, `stdout` and `stderr` - https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback

**Pull request merged**